### ISSUE TITLE
focal loss

### ISSUE THREAD

**Issue by auroua** [CasiaFan/tensorflow_retinanet#3]
https://github.com/CasiaFan/tensorflow_retinanet/blob/39d53cca8bc5bdb10f57614282ed2b568e349910/loss.py#L35

Why you multiply `onehot_labels` again in this line?

**Comment by zongfan2** [comment]
@auroua 
Eh... You are right it's a bug (I forgot it was used in pt calculate :-( ). Thanks very much for reporting such mistakes!

**Comment by zpge** [comment]
Is this a two-class version or multi-class version?

**Comment by github-luffy** [comment]
tf.nn.sigmoid_cross_entropy_with_logits()不是已经包括了负号嘛

**Comment by github-luffy** [comment]
论文的Focal Loss应该是单目标检测的吧
